Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -107,6 +107,12 @@ def tests
         ENV.delete(env)
       end
 
+      # Codespaces /tmp is mounted 755 which makes Ruby warn constantly.
```

Comment:
What warning is this? I've probably seen it before but can't recall exactly.

Reply:
It's maybe `775`.

Reply:
```
       expected block to not output to stderr, but output "/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/language/python.rb:10: warning: Insecure world writable dir /tmp/homebrew-tests-20221103-5773-8cjiu7/prefix/bin in PATH, mode 040756\n"
```

Reply:
756? That's a weird one...

I wonder if we can just set a 755 (or even a 700 is enough) on the directory created here: https://github.com/Homebrew/brew/blob/d863d333c7dfab4167c62887a8dfc821c8c3d9f3/Library/Homebrew/test/support/lib/startup/config.rb#L9

Reply:
Nope, a `chmod` here is insufficient.

Reply:
Ah I'm guessing it's still using 756 for subdirectories.

Probably some weird ACL going on (`getfacl /tmp`). `setfacl -k /tmp` probably works, not sure what the consequences on the rest of Codespaces would be though.

Reply:
I suppose this does raise the point, when creating `Keg::MUST_EXIST_DIRECTORIES` perhaps we should be doing `chmod "o-w"` on at least `bin` and `sbin` afterwards.